tests/Repository/EmailTemplateTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+ */
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusMailTemplatePlugin\Repository;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+final class EmailTemplateTest extends WebTestCase
+{
+    private $emailTemplateRepository;
+
+    protected function setUp(): void
+    {
+        self::bootKernel();
+
+        $this->emailTemplateRepository = self::$container->get('bitbag_sylius_mail_template_plugin.custom_repository.email_template');
+    }
+
+    public function test_return_used_template_email_types(): void
```

Comment:
Where are fixtures? 🙃

Reply:
Yes. Tests should be independent of application. We can discuss it in a call.

Reply:
By independend of application I mean, every test has to be able to be ran independently of application state. So in the beginning of integration test you have to create the database state (using fixtures and nelmio alice) and then perform testing code and assertions.

Reply:
Done